bin/remove_trailing_whitespace.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# removes all trailing whitespaces from *.sh and *.py files in this folder
+find . -type f \( -name "*.sh" -o -name "*.py" \) -exec sed -i 's/[[:space:]]\+$//' {} \;
```

Comment:
I'm only aware of one .py file in this repo, so I'm surprised it needed treatment

Reply:
Yes, you are right, I just assumed there could be more in the future, or if someone edits that file can run this script to remove trailing whitespaces. But it doesn't need treatment at this point.